mlflow/ml-package-versions.yml

```diff
@@ -313,6 +313,7 @@ spark:
   autologging:
     minimum: "3.0.0"
     maximum: "3.1.2"
+    requirements: ["scikit-learn"]
```

Comment:
Because in pyspark ml autologging tests, I use sklearn iris test dataset, so I add the dependency for sklearn here.